.github/workflows/validate-theme-entry.yml

```diff
@@ -0,0 +1,28 @@
+name: Validate Theme Entry
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    paths:
+      - community-css-themes.json
+
+jobs:
+  theme-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: "refs/pull/${{ github.event.number }}/merge"
+      - uses: actions/setup-node@v2
+      - run: npm install probe-image-size
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const script = require('./validation-theme.js');
+            const probe = require('probe-image-size');
```

Comment:
I'm not too familiar with github scoping. Is there any reason this import needs to be done here and passed in? Can we just import it from the validation-theme.js script?

Reply:
`require` here is a wrapper around the regular require.
We could also also pass in that wrapper.
<https://github.com/actions/github-script#use-npm-packages>